### ISSUE TITLE
question about blockingType

### ISSUE THREAD

**Issue by luanrp** [VDA5050/VDA5050#232]
When an AGV performs an action sequence on a specific node, [N1, H1,  N2, H2,  N3], (H means HARD, S means SOFT, N means NONE) the execution is as follows:

1. stop AGV and perform H1
2. After completing H1, perform H2 

My question is, which is correct next step, A or B?
A) Once H2 is completed, continue driving and perform N1 N2, N3 in parallel.
B) Once H2 is completed, perform N1 N2, N3; when all done, continue driving

(According to the definition of blockingType, I guess the answer should be A, AGV can start driving immediately), 
(according to the figure 15, the parallel execution list should be empty then continue to handle order; seems B is right)

**Comment by collani-bosch** [comment]
I think Figure 15 shows something different:
1. put N1 into parallel execution list
2. stop driving, execute N1, then execute H1
3. put N2 into parallel execution list
4. execute N2, then execute H2
5. put N3 into parallel execution list
6. execute N3
7. continue order handling
Question: Can the vehicle starts driving again during executing N3? I would say: yes. Can you rely on it: no. This depends on the specific vehicle. The action N3 cannot make any assumption whether the vehicle is driving or not. Otherwise the action shall not be executed as non blocking action or (if driving is necessary) as action on an edge.

**Comment by ConstantinUp** [comment]
With Version 3.0 there was a complete rework of the handling of action blocking types. Please review the new approach. If still unclear, feel free to open a new issue.
#374
